eva-accession-import/src/test/java/uk/ac/ebi/eva/accession/dbsnp/io/DbsnpClusteredVariantWriterTest.java

```diff
@@ -127,51 +125,21 @@ public void saveDifferentTaxonomies() throws Exception {
         assertEquals(secondClusteredVariant, accessions.get(1).getModel());
     }
 
-    @Test
-    public void failsOnDuplicateVariant() throws Exception {
```

Comment:
Even if not expecting to fail any more, I think it would be a good idea to have these more modular test methods. The last one looks a bit convoluted to me.

Reply:
Tests added again as part of the test class refactor commit.

Reply:
Tests added again as part of test class refactor.